doc/src/redis.rst

```diff
@@ -15,23 +15,57 @@ Configuration
 -------------
 
 Out of the box, Redis is set up with a couple of common default
-parameters and listens on *localhost* and the IP-addresses of the
-*ethsrv*-interface of your VM (See :ref:`networking` for details on this topic).
-
-listens on the *ethsrv* interface on port 6379.
-
-If you need to change the behaviour of Redis, you can put your
-additional configuration into :file:`/etc/local/redis/custom.conf`.
-
-Available configuration options can be found in the
-`official documentation <https://redis.io/topics/config>`_.
+parameters and listens on the IP-addresses of the *loopback* (localhost) and
+*ethsrv*-interfaces of your VM on port 6379 (See :ref:`networking`
+for details on this topic).
+
+If you need to change the behaviour of Redis, you define your redis
+configuration with the NixOS options ``flyingcircus.services.redis.*``.
```

Comment:
this should be `services.redis.settings` which replaces the `custom.conf` file. Adding an example for that would be helpful to make clear that it's different from `flyingcircus.services.redis.*`